src/main/kotlin/no/nav/familie/ef/iverksett/tilbakekreving/TilbakekrevingClient.kt

```diff
@@ -60,13 +59,8 @@ class TilbakekrevingClient(@Qualifier("azure") restOperations: RestOperations,
                              HttpHeaders().apply { accept = listOf(MediaType.APPLICATION_PDF) })
     }
 
-    fun opprettBehandling(opprettTilbakekrevingRequest: OpprettTilbakekrevingRequest): String {
-        val response: Ressurs<String> = postForEntity(opprettTilbakekrevingUri, opprettTilbakekrevingRequest)
-        return response.getDataOrThrow()
-    }
-
-    fun opprettBehandlingManuelt(request: OpprettManueltTilbakekrevingRequest) {
```

Comment:
Hva er forskjellen mellom disse? Og trenger vi ikke manuell opprettelse noe lengre?

Reply:
Manuell opprettelse gjøres rett fra ef-sak nå. Det er ikke nødvendig å gå gjennom iverksett for det kallet.